wrapper/pycc/src/qcc_db/ccQuadric.cpp

```diff
@@ -65,7 +65,7 @@ void define_ccQuadric(py::module &m)
                 [](CCVector2 minCorner,
                    CCVector2 maxCorner,
                    const py::sequence eq,
-                   const Tuple3ub *dims = nullptr,
+                   const CCCoreLib::SquareMatrix *localOrientation = nullptr,
```

Comment:
the corresponding `"dims"_a` a few lines down also have to be updated

Reply:
Oops, I just saw the documentation above also 😆